src/figuresClasses.ts

```diff
@@ -1,19 +1,66 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
Better to create enum for those
![image](https://github.com/mate-academy/ts_figures_classes/assets/139622699/8f8b55c2-f0a3-4188-bd33-ea6b3f61f1e3)

Reply:
@AntonHerasymov it's only about `enum`